### PR TITLE
identity tabs

### PR DESCRIPTION
**Pull request by wuyuansushen**

### CODE REVIEW

File: docs/management/sentinel.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ One set of instances is called `mymaster`, and the other `resque`.
 
 The meaning of the arguments of `sentinel monitor` statements is the following:
 
-    sentinel monitor <master-group-name> <ip> <port> <quorum>
+    sentinel monitor <master_name> <ip> <port> <quorum>
```
Comment (enjoy-binbin) [inline docs/management/sentinel.md:120]: i see in sentinel.conf, the example is:
```
# sentinel monitor <master-name> <ip> <redis-port> <quorum>
```

so let change it to this?
```suggestion
    sentinel monitor <master-name> <ip> <port> <quorum>
```

also, please scan the sentinel.md file, i see there are some `master-group-name` word, i think we can change it both

**Review by enjoy-binbin — COMMENTED** [review]
also, please scan the sentinel.md file (this one), i see there are some master-group-name word, i think we can change it both

**Comment by wuyuansushen** [comment]
> also, please scan the sentinel.md file (this one), i see there are some master-group-name word, i think we can change it both

OK, I will continue my work.

**Comment by wuyuansushen** [comment]
Closed. I will open another PR to this change.

**Pull request closed without merging**